scripts/depcheck/missing.ts

```diff
@@ -52,6 +52,7 @@ const IGNORE_MATCHES_IN_PACKAGE = {
     'sass',
     'stylus',
     'tailwindcss',
+    'enquirer',
```

Comment:
I think you should add it to `@nrwl/angular` `package.json`

Reply:
ok no problem